app/Services/Account/Settings/ExportAccount.php

```diff
@@ -0,0 +1,1366 @@
+<?php
+
+namespace App\Services\Account\Settings;
+
+use App\Models\User\User;
+use App\Services\BaseService;
+use App\Models\Account\Account;
+use App\Models\Contact\Document;
+use Illuminate\Support\Facades\DB;
+use App\Exceptions\NoAccountException;
+
+class ExportAccount extends BaseService
+{
+    protected $sql;
```

Comment:
There are 2 problems here:
- first you should not use a class variable here. It's better to use local variable in `execute` function and pass it to each functions.
- this could link to a memory limit problem, because this `$sql` variable could be huge. We should find a way to write each line in a file directly, so there is no risk of memory limit.

Reply:
I've thought about that.
Let me check how I could enhance it.

Reply:
When I export a sample account (using `artisan setup` command), the SQL can go up to 3.5Mb of data. It won't be good in production on large accounts.